hles/extraction/src/main/scala/org/broadinstitute/monster/dap/FilterDirective.scala

```diff
@@ -23,4 +23,14 @@ object FilterOps {
   * Represents a typesafe REDCap filterLogic directive.
   * Their API expects directives to be of the form [field][op][value]
   */
-case class FilterDirective(field: String, operation: FilterOp, comparand: String)
+case class FilterDirective(field: String, operation: FilterOp, comparand: String) {
+
+  // wrap any comparands with a space in quotes
+  def spaceEscapedComparand: String = {
+    comparand match {
+      case comparandWithSpace: String if comparandWithSpace.contains(" ") =>
+        s""""${comparandWithSpace}""""
```

Comment:
`s"\"${comparandWithSpace}\""` runs into an eight-year standing (!!!) bug with scala where character escaping is completely broken in s-strings.

Reply:
Great find, can you add any color to this?

Reply:
https://github.com/scala/bug/issues/6476 bug info is here, dating back to october 2012